src/main/java/org/frc5687/rapidreact/Constants.java

```diff
@@ -18,7 +18,7 @@
         //Swerve modules renamed from positions to directions
         //I did it ya'll happy? :) Where's my golden star?
         public static final Translation2d NORTH_EAST = new Translation2d(WIDTH / 2.0, LENGTH / 2.0);
-        public static final double NORTH_EAST_OFFSET = 0; // radians
+        public static final double NORTH_EAST_OFFSET = -0.349066; // radians
```

Comment:
Again, I was told that the hardware has been modified so that we do not need an offset. I need someone to confirm this.